services/src/main/java/org/broadleafcommerce/payment/service/gateway/DefaultPayPalCheckoutTransactionService.java

```diff
@@ -358,6 +358,20 @@ protected Capture capturePayment(Authorization auth, PaymentRequest paymentReque
         return response.getCapture();
     }
 
+    @Nullable
+    private String getStringAmount(@Nullable MonetaryAmount amount) {
+        return Objects.toString(normalizePrice(amount), null);
+    }
```

Comment:
`refundPayment(Capture capture, PaymentRequest paymentRequest)` should also call this method